spec/src/main/doc/core/implementation.asciidoc

```diff
@@ -730,7 +494,7 @@ A bean constructor may have any number of parameters. All parameters of a bean c
 
 === Injected fields
 
-An _injected field_ is a non-static, non-final field of a bean class or of any Java EE component class supporting injection.
+An _injected field_ is a non-static, non-final field of a bean class.
```

Comment:
After check it was the only place. Other deal with automatic injection (when these unmanaged object have manual injection) or automatic check or SPI events firing that are not related to these objects as well.